sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -658,6 +671,125 @@ def __repr__(self):
         )[:1024]
 
 
+class SentenceAspect(DictMixin):
+    """SentenceAspect contains the related opinions, predicted sentiment,
+    confidence scores and other information about an aspect of a sentence.
+    An aspect of a sentence is a key component of a sentence, for example
+    in the sentence "The food is good", "food" is an aspect.
+
+    :ivar str text: The aspect text.
+    :ivar str sentiment: The predicted Sentiment for the aspect. Possible values
+        include 'positive', 'mixed', and 'negative'.
+    :ivar confidence_scores: The sentiment confidence score between 0
+        and 1 for the aspect for 'positive' and 'negative' labels. It's score
+        for 'neutral' will always be 0
+    :vartype confidence_scores:
+        ~azure.ai.textanalytics.SentimentConfidenceScores
+    :ivar opinions: All of the opinions in the sentence related to this aspect.
+    :vartype opinions: list[~azure.ai.textanalytics.AspectOpinion]
+    :ivar int offset: The aspect offset from the start of the sentence.
```

Comment:
this will release with the encoding parameter, right? just want to make sure if we're exposing offset/length

Reply:
There won't be the encocding parameter for this release. We decided in scrum that since this is a preview, we're ok with adding back PII and offset, length, and count without the encoding parameter since the next TA release will have it. (Thaminda has said that it will be in PPE shortly after the service starts deployment on the 16th)